test/units/TEST-07-PID1.concurrency.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# shellcheck disable=SC2016
+set -eux
+set -o pipefail
+
+# shellcheck source=test/units/test-control.sh
+. "$(dirname "$0")"/test-control.sh
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
+
+cat >/etc/systemd/system/concurrency1.slice <<EOF
```

Comment:
Please always write configs to /run/ in integration tests, so that there are no leftovers around in the image even when things fail